docs/marionette.collectionview.md

```diff
@@ -22,7 +22,14 @@ in the DOM. This behavior can be disabled by specifying `{sort: false}` on initi
 * [CollectionView's children](#collectionviews-children)
   * [CollectionView's `buildChildView`](#collectionviews-buildchildview)
   * [CollectionView's `addChildView`](#collectionviews-addchildview)
+  * [CollectionView: Retrieve Child Views](#collectionviews-retrievechildview)
+    * [CollectionView's: `findByCid`](#collectionviews-retrievechildview-findbycid)
```

Comment:
I think the menu items should specify that it's the children somehow I think.. Otherwise just skimming the contents you would assume you could `myCollectionView.findByCid(1)`